src/main/resources/icons/files/semantic-release.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
```

Comment:
Please add a default width and height of 16px and add a better color, this one is too dark.

Reply:
Done. Is it better?